daily_milestone.go

```diff
@@ -158,34 +158,57 @@ func getMilestones(baseURL string, token string, projectID string) ([]simpleMile
 func createMilestoneData(advance int, timeInterval string) []simpleMilestone {
 	today := time.Now().Local()
 	list := []simpleMilestone{}
-	for i := 0; i < advance; i++ {
-		if timeInterval == "daily" {
+
+	if timeInterval == "daily" {
+		for i := 0; i < advance; i++ {
 			date := today.AddDate(0, 0, i).Format("2006-01-02")
 			milestone := simpleMilestone{}
 			milestone.Title = date
 			milestone.DueDate = date
 			list = append(list, milestone)
-		} else if timeInterval == "weekly" {
-			today = today.AddDate(0, 0, 7)
-			year, week := today.ISOWeek()
-			milestoneYear := strconv.Itoa(year)
-			milestoneWeek := strconv.Itoa(week)
+		}
+	} else if timeInterval == "weekly" {
+		newDate := today
+		day := newDate.Weekday()
+		year, week := newDate.ISOWeek()
+		x := day - day // initializing x to zero in time.Time
+		for i := 0; i < advance; i++ {
+			for j := x; j < 7; j++ {
```

Comment:
You are not using j, so why the for loop?

Reply:
that's to set the day to Sunday 

https://play.golang.org/p/yGjYNEmJ1G

Reply:
Ah I see. You are using the break to do that. Interesting, but then you can simplify it I think. Let me check that. For now that's fine.